plugins/modules/aci_l4l7_device.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_device
+short_description: Manage L4-L7 Devices (vns:LDevVip)
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Devices.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  name:
+    description:
+    - The name of the L4-L7 device.
+    type: str
+    aliases: [ device, device_name, logical_device_name ]
```

Comment:
Shouldn't we add an extra alias? `logical_device`?
```suggestion
    aliases: [ device, device_name, logical_device,  logical_device_name ]
```